dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -64,16 +65,60 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
+    void RegisterUnpackagedApplicationHelper(
+        const winrt::guid& remoteId,
+        _Out_ wil::unique_cotaskmem_string &unpackagedAppUserModelId)
+    {
+        auto coInitialize = wil::CoInitializeEx();
+
+        auto notificationPlatform{ wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER) };
+
+        wil::unique_cotaskmem_string processName;
+        THROW_IF_FAILED(GetCurrentProcessPath(processName));
+
+        THROW_IF_FAILED(notificationPlatform->RegisterFullTrustApplication(processName.get(), remoteId, &unpackagedAppUserModelId));
+    }
+
+    winrt::hresult CreateChannelWithRemoteIdHelper(const winrt::guid& remoteId, ChannelDetails& channelInfo) noexcept try
+    {
+        wchar_t appUserModelId[APPLICATION_USER_MODEL_ID_MAX_LENGTH] = {};
+        UINT32 appUserModelIdSize{ ARRAYSIZE(appUserModelId) };
+
+        THROW_IF_FAILED(GetCurrentApplicationUserModelId(&appUserModelIdSize, appUserModelId));
+
+        THROW_HR_IF(E_INVALIDARG, (appUserModelIdSize > APPLICATION_USER_MODEL_ID_MAX_LENGTH) || (appUserModelIdSize == 0));
+
+        HRESULT operationalCode{};
+        ABI::Windows::Foundation::DateTime channelExpiryTime{};
+        wil::unique_cotaskmem_string channelId;
+        wil::unique_cotaskmem_string channelUri;
+
+        THROW_IF_FAILED(PushNotifications_CreateChannelWithRemoteIdentifier(
+            appUserModelId,
+            remoteId,
+            &operationalCode,
+            &channelId,
+            &channelUri,
+            &channelExpiryTime));
+
+        THROW_IF_FAILED(operationalCode);
+
+        winrt::copy_from_abi(channelInfo.channelExpiryTime, &channelExpiryTime);
+        channelInfo.appUserModelId = winrt::hstring{ appUserModelId };
+        channelInfo.channelId = winrt::hstring{ channelId.get() };
+        channelInfo.channelUri = winrt::hstring{ channelUri.get() };
+
+        return S_OK;
+    }
+    CATCH_RETURN()
+
     winrt::IAsyncOperationWithProgress<winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelResult, winrt::Microsoft::Windows::PushNotifications::PushNotificationCreateChannelStatus> PushNotificationManager::CreateChannelAsync(const winrt::guid &remoteId)
     {
         try
         {
             THROW_HR_IF(E_INVALIDARG, (remoteId == winrt::guid()));
 
-            // API supports channel requests only for packaged applications for v0.8 version
-            THROW_HR_IF(E_NOTIMPL, !AppModel::Identity::IsPackagedProcess());
-
-            auto cancellation{ co_await winrt::get_cancellation_token() };
+			auto cancellation{ co_await winrt::get_cancellation_token() };
```

Comment:
Some weird indenting going on here

Reply:
Generally, make a pass and do spaces instead of tabs, four-space indent in a scope.  @ChrisGuzak was looking into getting a `.editorconfig` for use in projects.